java/test/jmri/jmrit/roster/RosterTest.java

```diff
@@ -335,8 +335,8 @@ public Roster createTestRoster() throws IOException, FileNotFoundException {
         FileUtil.createDirectory(rosterDir);
 
         File f = new File(rosterDir, "rosterTest.xml");
-        // remove existing roster if its there
-        f.delete();
+        // File should never be there is TemporaryFolder working
+        if (f.exists()) Assert.fail("rosterTest.xml in "+rosterDir+" already present: "+f);
```

Comment:
Mostly force of habit.  In this case, I'm using "fail" to drive home that it really is a failure of the test setup. To me (again, due to habit) the `if (..) Assert.fail("why");` reads clearer in that case: If this is true, the test setup has failed.

The alternative form I typically use would be `assertTrue(!f.exists(), "reason");` if it was a regression test of the Roster class itself.